app/src/main/java/com/conupods/OutdoorMaps/Models/PointsOfInterest/Photo.java

```diff
@@ -2,20 +2,20 @@
 
 public class Photo
 {
-    private String photo_reference;
+    private String photoReference;
 
     private String width;
 
     private String height;
 
-    public String getPhoto_reference ()
+    public String getPhotoReference()
     {
-        return photo_reference;
+        return photoReference;
     }
 
-    public void setPhoto_reference (String photo_reference)
+    public void setPhotoReference(String photoReference)
```

Comment:
Do not change the name of this attribute, it must match the attribute name that is within the JSON response object. Revert change.